gateway-server/src/test/java/org/apache/knox/gateway/util/ServiceDefinitionsLoaderTest.java

```diff
@@ -18,35 +18,45 @@
 package org.apache.knox.gateway.util;
 
 import org.apache.knox.gateway.deploy.ServiceDeploymentContributor;
+import org.apache.knox.gateway.service.definition.ServiceDefinition;
+import org.apache.knox.gateway.service.definition.ServiceDefinitionComparator;
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.common.collect.Ordering;
```

Comment:
Think this is the only place guava comes in

Reply:
Yes, it is...